pkg/3scale/amp/component/backend.go

```diff
@@ -422,6 +464,9 @@ func (backend *Backend) buildBackendListenerEnv() []v1.EnvVar {
 			v1.EnvVar{Name: "CONFIG_LISTENER_PROMETHEUS_METRICS_ENABLED", Value: "true"},
 		)
 	}
+	if backend.Options.RedisTLSEnabled {
```

Comment:
we already have these in buildBackendCommonEnv - common in my understanding applies to both, worker and listener - is this really required?